clients/react-adapter.rst

```diff
@@ -1,2 +1,77 @@
 React Adapter
 =============
+
+An HTTPlug adapter for the `React Http client`_.
+
+Installation
+------------
+
+To install the React adapter, which will also install React itself (if it was
+not yet included in your project), run:
+
+.. code-block:: bash
+
+    $ composer require php-http/react-adapter
+
+Usage
+-----
+
+The React client adapter needs a :ref:`message factory <message-factory>` in
+order to to work::
+
+    use Http\Adapter\React\Client;
+
+    $client = new Client($messageFactory);
+
+For simplicity, all the required objects are built internally by default:
```

Comment:
s/built internally by default/instantiated automatically if not explicitly specified/ ?